gammapy/data/event_list.py

```diff
@@ -12,25 +12,30 @@
 from astropy.table import vstack as vstack_tables
 from ..utils.energy import EnergyBounds
 from ..utils.scripts import make_path
+from ..image import SkyImage
```

Comment:
We have to be careful with "circular imports", and try to have a clear hierarchy of lower-level and higher-level sub-packages in Gammapy. `gammapy.data` is lower level than `gammapy.image`, so I'm surprised that your import of `SkyImage` here in `gammapy.data` doesn't give an error.

Still: can you please make this a "delayed import" by moving the import statement into the functions or methods where you need it? In this case in the `EventListLAT.plot_image` method.

Reply:
Done!